### ISSUE TITLE
Accounts dropping Like Flies

### ISSUE THREAD

**Issue by BotMassive** [finndev/PokeBuddy#1110]
Well, there we go, the ban wave caught up with me and my main account is banned. Level 34, several Pokemon 2500-3000+ CP. 

Hours and hours of farming gone down the shitta :(

I had a feeling it would happen, a friend of mine who started botting just before me got a ban yesterday, the usual thing, ""Failed to get game data from the server".

To make it that bit worse, my secondary account of level 33 got a ban 2 days ago as well.

At least one other account of mine has been deleted in the last 2 days.

How many accounts do I have you ask, well...look at my username :)

So be warned, even if you've stopped botting you can still get banned. If you have a high level account worth keeping and your're still botting you are dicing with death, well, death of your account.

Ciao

**Comment by Koitah** [comment]
![sem titulo](https://cloud.githubusercontent.com/assets/20827081/17651564/aac5c88e-6261-11e6-9709-57acbea426bd.jpg)

**Comment by Koitah** [comment]
0 ban in this account ;)

**Comment by BotMassive** [comment]
Lucky you, share your methods please to stay clear of bans.

I've used VPN since IP's started getting banned, but that didn't stop the accounts from being banned or deleted.

Also kept the botting time to 1-2 hours max.

Looking at the distance you walked you must be at a walk speed of 15-20 km/h?

Maybe going at 40 km/h is what got my accounts flagged.

It would be good to have a definitive list on how to stay safe.

**Comment by IamDeric** [comment]
0 ban 😎 , after i read some articles about the ban wave, i stop botting, but before that my 2nd (pure bot) and 3rd (pure bot) account with lvl 30 each (give to my sister and his bf) i stop it ahead for 5days and never log in. For my main account with lvl 34 (lvl 23 start botting) and half way to 35, after i hit lvl 34 and were getting alot of issue on botting i only do farm pokestop and uncheck everything with 5km/p. After i hit 350k stardust (to catch up with the 800k stardust that i use for powering up my main pokemons) i totally stop botting and playing manually in my phone and i never get any error or ban at all.

**Comment by Brightblazer** [comment]
You are wrong cause band seems to affect and other players even if they don't Notre my gf LVL 22 got banned we were using fly gps ... only I got banned in first acc before three days LVL 27 full voted though my second acc still alive only manually used with GPS but I can't predicted why will happen now

**Comment by Brightblazer** [comment]
We didn't use Pokemon in gyms at all
